tests/test_views.py

```diff
@@ -116,5 +122,49 @@ def test_response_404_when_none_specified(self):
             content = json.loads(response.content.decode('utf-8'))
             self.assertCountEqual({'message': 'No checks found', 'error': 404}, content)
 
+    @override_settings(WATCHMAN_LOGIN=False)
+    def test_login_not_required(self):
+        # Reload settings - and all dependent modules - from scratch
+        reload(sys.modules['watchman.settings'])
+        reload(sys.modules['watchman.decorators'])
+        reload(sys.modules['watchman.views'])
```

Comment:
Ah, makes sense why it's needed now.

I'd be +1 to pulling it out into a small function (generally I like doing that the third time I find myself copy / pasting some code). I wouldn't be surprised if more tests are added which require the `reload` as well (eg. once this and #31 are merged in, we may want to test the `login_required` functionality on that view as well.